### PR TITLE
Gaming

### PR DESCRIPTION
**Pull request by HellBringer419**
Working on issue #151 this is WIP#3

This PR includes:
* remove the click function from images
* correcting the package for PolyChromatic

Reviewers:
@fossfreedom

### CODE REVIEW

File: data/gaming.html [hunk 1]
```diff
@@ -41,9 +41,9 @@ <h2><a
                                                 class="material-icons">help</span></a>&zwnj;Lutris&zwnj;</h2>
                                     <div class="card card-block">
                                         <div class="card-image text-md-center">
-                                            <a href="img/logos/lutris.png" data-toggle="lightbox"><img
+                                            <img
```
Comment (fossfreedom) [inline data/gaming.html:44]: Stylistically lets have the img and the first line together rather than a new line.

File: data/gaming.html [hunk 2]
```diff
@@ -74,9 +74,9 @@ <h2><a onclick="cmd('link?https://help.steampowered.com/en/')"><span
                                                 class="material-icons">help</span></a>&zwnj;Steam&zwnj;</h2>
                                     <div class="card card-block">
                                         <div class="card-image text-md-center">
-                                            <a href="img/logos/steam.png" data-toggle="lightbox"><img
+                                            <img
```
Comment (fossfreedom) [inline data/gaming.html:77]: similar comment as above

File: data/gaming.html [hunk 3]
```diff
@@ -117,9 +117,9 @@ <h2><a onclick="cmd('link?https://deltarune.com/')"><span
                                     </h2>
                                     <div class="card card-block">
                                         <div class="card-image text-md-center">
-                                            <a href="img/screenshots/DELTARUNE.jpg" data-toggle="lightbox"><img
+                                            <img
```
Comment (fossfreedom) [inline data/gaming.html:120]: and so on for all the remaining img changes

**Review by fossfreedom — CHANGES_REQUESTED** [review]

File: data/gaming.html (hunk 1 above)
Comment (HellBringer419) [reply data/gaming.html]: Sorry didn't see these earlier.
formatted.

File: data/gaming.html (hunk 2 above)
Comment (HellBringer419) [reply data/gaming.html]: formatted

File: data/gaming.html (hunk 3 above)
Comment (HellBringer419) [reply data/gaming.html]: formatted

**Comment by fossfreedom** [comment]
+1 - many thanks - the snap in edge will be update automatically in the next few hours

**Pull request merged**